geode-assembly/src/acceptanceTest/java/org/apache/geode/client/sni/DualServerSNIAcceptanceTest.java

```diff
@@ -65,11 +65,6 @@
   private static final URL DOCKER_COMPOSE_PATH =
       SingleServerSNIAcceptanceTest.class.getResource("docker-compose.yml");
 
-  // Docker compose does not work on windows in CI. Ignore this test on windows
-  // Using a RuleChain to make sure we ignore the test before the rule comes into play
-  @ClassRule
-  public static TestRule ignoreOnWindowsRule = new IgnoreOnWindowsRule();
-
   @ClassRule
```

Comment:
Make your DockerComposeRule just a Rule, not a ClassRule.

Reply:
I need it to be a `@ClassRule` so I can access it from a (`static`) `@BeforeClass`

Reply:
ugh

Reply:
@upthewaterspout why does it need to be a Rule instead of a ClassRule?  I'm doing the same in my PR (https://github.com/apache/geode/pull/4938) and would like to avoid problems, but this really speeds things up if you have multiple tests in the class.